ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -24,12 +24,108 @@ periodics:
   labels:
     master: ci.openshift.redhat.com
   name: azure_build_base_image_rhel
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  labels:
+    master: ci.openshift.redhat.com
+  name: azure-build-base-image-centos
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --artifact-dir=$(ARTIFACTS)
+      - --secret-dir=/usr/local/e2e-secrets/azure
+      - --template=/usr/local/build-base-image
+      - --target=build-base-image
+      command:
+      - ci-operator
+      env:
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: openshift-openshift-ansible-release-3.10.yaml
+            name: ci-operator-configs
+      - name: JOB_NAME_SAFE
+        value: build-base-image
+      - name: DEPLOY_OS
+        value: centos
+      image: ci-operator:latest
+      imagePullPolicy: Always
+      name: ""
+      resources: {}
+      volumeMounts:
+      - mountPath: /usr/local/build-base-image
+        name: job-definition
+        subPath: build-base-image.yaml
+      - mountPath: /usr/local/e2e-secrets/azure
+        name: cluster-secrets-azure-file
+    serviceAccountName: ci-operator
+    volumes:
+    - configMap:
+        name: prow-job-build-base-image
+      name: job-definition
+    - name: cluster-secrets-azure-file
+      projected:
+        sources:
+        - secret:
+            name: cluster-secrets-azure-file
   run_after_success:
   - agent: kubernetes
     cron: ""
     decorate: true
     interval: 24h
-    name: azure-build-node-image-rhel-310
+    name: azure-build-node-image-centos
```

Comment:
azure-build-node-image-centos-310

Reply:
I see you use this elsewhere. Then CONFIG_SPEC needs to change here.

Reply:
done